src/NumericalAlgorithms/LinearAlgebra/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY LinearAlgebra)
+
+set(LIBRARY_SOURCES
+    FindGeneralizedEigenvalues.cpp
```

Comment:
make indentation consistent in this file

Reply:
indent two characters not four